build.gradle

```diff
@@ -16,6 +16,13 @@ plugins {
   id "com.github.spotbugs" version "4.0.1"
 }
 
+idea {
+    module {
+        downloadJavadoc = false
+        downloadSources = false
+    }
+}
+
```

Comment:
is this needed since also in `allprojects` block below?

Reply:
Actually not need, I'll remove it.